src/bench/bench.cpp

```diff
@@ -63,6 +85,11 @@ void benchmark::BenchRunner::RunAll(const Args& args)
 
     std::vector<ankerl::nanobench::Result> benchmarkResults;
     for (const auto& p : benchmarks()) {
+
+        if (!(p.second.second & args.priority_level)) {
```

Comment:
Will expand the diff a bit, but wdyt about using structured bindings for iterating over `benchmarks()` and unpacking the second pair (as you can't do nested structured bindings)? `p.second.second` could benefit from some improved readability.

Reply:
sure, give me a second.